security/avira.ksy

```diff
@@ -0,0 +1,59 @@
+meta:
+  id: qua
+  file-extension: qua
+  endian: le
+  title: Avira Antivirus quarantine file parser
+  license: MIT
+  ks-version: 0.9
+doc: |
+  Creator: Florian Bausch, ERNW Research GmbH, https://ernw-research.de
+  License: MIT
+  https://github.com/ernw/quarantine-formats/blob/master/docs/Avira_Antivirus.md
+  Avira Antivirus quarantine files are usually stored under /ProgramData/Avira/Antivirus/INFECTED.
+  The file extension usually is .qua.
+seq:
+  - id: magic
+    size: 16
+    contents: ["AntiVir Qua", 0x00, 0x00, 0x00, 0x00, 0x00]
+  - id: malicious_offset
+    type: u4
```

Comment:
Why isn't it used?

Reply:
Because the parser also works without using it. I assume the offset is stored there for a quicker lookup of the encrypted malware file, e.g. when restoring the file.

Reply:
or maybe it is done for compatibility and some other fields are possible.